addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/ActiveScanJobDialog.java

```diff
@@ -142,45 +148,73 @@ public ActiveScanJobDialog(ActiveScanJob job) {
             allstrengths.add(JobUtils.strengthToI18n(at.name()));
         }
 
-        this.addComboField(1, DEFAULT_STRENGTH_PARAM, allstrengths, strengthName);
+        this.addComboField(tabIndex, DEFAULT_STRENGTH_PARAM, allstrengths, strengthName);
 
-        this.addPadding(1);
+        this.addPadding(tabIndex);
 
         List<JButton> buttons = new ArrayList<>();
         buttons.add(getAddButton());
         buttons.add(getModifyButton());
         buttons.add(getRemoveButton());
 
-        this.addTableField(2, getRulesTable(), buttons);
+        String tagRuleThresholdName =
+                JobUtils.thresholdToI18n(
+                        job.getData()
+                                .getPolicyDefinition()
+                                .getAlertTagRule()
+                                .getThreshold()
+                                .name());
+        if (tagRuleThresholdName.isEmpty()) {
+            tagRuleThresholdName = JobUtils.thresholdToI18n(AlertThreshold.MEDIUM.name());
+        }
+        String tagRuleStrengthName =
+                JobUtils.strengthToI18n(
+                        job.getData().getPolicyDefinition().getAlertTagRule().getStrength().name());
+        if (tagRuleStrengthName.isEmpty()) {
+            tagRuleStrengthName = JobUtils.strengthToI18n(AttackStrength.MEDIUM.name());
+        }
+        this.addComboField(
+                ++tabIndex, TAG_RULE_THRESHOLD_PARAM, allthresholds, tagRuleThresholdName);
+        this.addComboField(tabIndex, TAG_RULE_STRENGTH_PARAM, allstrengths, tagRuleStrengthName);
+        this.addTableField(
+                tabIndex,
+                getIncludedAlertTagsTable(),
+                List.of(getAddIncludedAlertTagButton(), getRemoveIncludedAlertTagButton()));
+        this.addTableField(
+                tabIndex,
+                getExcludedAlertTagsTable(),
+                List.of(getAddExcludedAlertTagButton(), getRemoveExcludedAlertTagButton()));
+
+        this.addTableField(++tabIndex, getRulesTable(), buttons);
```

Comment:
I'd suggest adding this to the base class, e.g. `protected int addTagRuleTab(int tabIndex, List<String> allthresholds, List<String> allstrengths)`.